.agents/skills/trellis-meta/references/platform-files/platform-map.md

```
@@ -84,3 +88,20 @@ Platform ecosystems change, and user projects may already be customized. If this
 - Judge behavior by the read rules currently written in the agent file.
 
 Do not delete a custom file just because it is not listed in this path table.
+
+### `.omp/` — Oh My Pi (OMP)
+
+Extension-backed platform. OMP native provider auto-discovers all subdirectories.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)